pandas/tests/scalar/test_timestamp.py

```diff
@@ -1136,6 +1136,26 @@ def test_timestamp(self):
                 dt = ts.to_pydatetime()
                 assert dt.timestamp() == ts.timestamp()
 
+    def test_replace(self):
```

Comment:
this can't be the right place, we have lots of replace tests

Reply:
not in test_timestamp... I'll take another look.

Reply:
Looks like its in test_timezones.